algo/src/main/java/org/neo4j/graphalgo/impl/walking/WalkResult.java

```diff
@@ -0,0 +1,17 @@
+package org.neo4j.graphalgo.impl.walking;
+
+import org.neo4j.graphdb.Path;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WalkResult {
+    public Path path;
+    public List<Long> nodes;
```

Comment:
Can we name this `nodeIds`? I can always do it after merging - just would make it more consistent with what we're doing elsewhere

Reply:
Sure